### PR TITLE
Refining the template notebook

### PR DESCRIPTION
**Pull request by cansavvy**
## Purpose:

Trying out a manner of commenting what material is part of the template and what is new. (I'm finding while reviewing #151 that this will be annoying for reviewers So I have some `<!-- comments-->` to try to mark where material is added and these are mean to be left.)

Also we noted some places where the template notebook was not adhering to the style guide (thanks @cbethell )

Also noticed the screenshot `download-analysis-folder` actually wasn't cropped at the informative part so I redid it and updated the src links to that commit. 
## Strategy

There are also two types of comments `<!-- -->` are meant to be left (but I hope they don't confuse the users?) And `{{}}` should be replaced with the proper information.

### CODE REVIEW

File: template/template_example.Rmd [hunk 1]
```diff
@@ -62,42 +66,46 @@ if (!dir.exists(results_dir)) {
...
+<img src="https://raw.githubusercontent.com/AlexsLemonade/refinebio-examples/master/template/screenshots/download-email.png" width=500>  
 
 It make take a few minutes for the dataset to process.
 You will get an email when it is ready. 
 
 ## About the dataset we are using for this example
+<!-- ADD PATH AND SUMMARY -->
 
-For this example analysis, we will use this [<<BRIEF DATASET NAME>>](https://alexslemonade.github.io/refinebio-examples/ <<RELATIVE PATH TO THIS ANALYSIS FROM REFINEBIO-EXAMPLES>> )..
+For this example analysis, we will use this [<!--BRIEF DATASET NAME-->](https://alexslemonade.github.io/refinebio-examples/ <!--RELATIVE PATH TO THIS ANALYSIS FROM REFINEBIO-EXAMPLES--> )..
```
Comment (jashapiro) [inline template/template_example.Rmd:89]: Did you test this? I am not sure if you can put a comment inside a markdown url... 

```suggestion
For this example analysis, we will use this [<<BRIEF DATASET NAME>>](https://alexslemonade.github.io/refinebio-examples/ <!--RELATIVE PATH TO THIS ANALYSIS FROM REFINEBIO-EXAMPLES--> )..
```

File: template/template_example.Rmd [hunk 2]
```diff
@@ -108,32 +116,33 @@ Your new analysis folder should contain:
...
-<img src="screenshots/analysis-folder-structure.png" width=400>
+<img src="https://raw.githubusercontent.com/AlexsLemonade/refinebio-examples/master/template/screenshots/analysis-folder-structure.png" width=400>
 
 In order for our example here to run without a hitch, we need these files to be in these locations so we've constructed a test to check before we get started with the analysis. 
 Run this chunk to double check that your files are in the right place. 
 
 ```{r}
 # Check if the gene expression matrix file is in the data directory stored as `data_dir`
-file.exists(file.path(data_dir, <<DATA_ACCESSION FILENAME>>))
+file.exists(file.path(data_dir, <!--DATA_ACCESSION FILENAME-->))
```
Comment (jashapiro) [inline template/template_example.Rmd:126]: I think you want to leave this (and the next one) as it was (pretty sure html comments don't work in code blocks)
```suggestion
file.exists(file.path(data_dir, <<DATA_ACCESSION FILENAME-->))
```

Comment (cansavvy) [reply template/template_example.Rmd:126]: I think i missed some of these because I switch strategies.

**Comment by jashapiro** [comment]
When reviewing #166, I noticed that the use of `<<TEMPLATE>>` sometimes results in rendering oddities as the angle brackets can be interpreted as (broken) HTML tags. I suggest we switch to something that html does not use; perhaps `{{TEMPLATE}}`?

File: template/template_example.Rmd [hunk 3]
```diff
@@ -11,7 +11,7 @@ output:
 
 # Purpose of the analysis:
 
-<<WHAT DOES THIS ANALSIS DO AND WHEN WOULD YOU USE IT?>>
+<!--WHAT DOES THIS ANALYSIS DO AND WHEN WOULD YOU USE IT?-->
```
Comment (cbethell) [inline template/template_example.Rmd:14]: So the idea here would be to add context based on this prompt but also leave this prompt in the notebook above the context once added?

Comment (cansavvy) [reply template/template_example.Rmd:14]: Yes. That's the idea. That way when we review things we know where to expect added content.

Comment (cbethell) [reply template/template_example.Rmd:14]: Gotcha 👍

**Pull request merged**